cdap-app-fabric/src/main/java/io/cdap/cdap/support/handlers/SupportBundleHttpHandler.java

```diff
@@ -40,11 +44,15 @@
 @Path(Constants.Gateway.API_VERSION_3)
 public class SupportBundleHttpHandler extends AbstractAppFabricHttpHandler {
 
-  private static final Logger LOG = LoggerFactory.getLogger(SupportBundleHttpHandler.class);
   private final SupportBundleService bundleService;
+  private final AccessEnforcer accessEnforcer;
+  private final AuthenticationContext authenticationContext;
 
   @Inject
-  SupportBundleHttpHandler(SupportBundleService supportBundleService) {
+  SupportBundleHttpHandler(AccessEnforcer accessEnforcer, AuthenticationContext authenticationContext,
```

Comment:
nit: You can probably use [ContextAccessEnforcer](https://github.com/cdapio/cdap/blob/dd815de91824e87e028902b80a630e2de4d30933/cdap-security-spi/src/main/java/io/cdap/cdap/security/spi/authorization/ContextAccessEnforcer.java) to save yourself from needing to manually import AccessEnforcer and AuthenticationContext.